main/process-tx.zkasm

```diff
@@ -68,9 +70,34 @@ endCheckChainId:
                                         :CALL(isColdAddress) ; add tx.origin to touched addresses
         0                               :MSTORE(depth) ; Initial depth is 0
 
-;; Set gasPrice global var
+;; Set gasPrice global var depending on effectivePercentage [0-255] -> txGasPrice = Floor((gasPrice * (effectivePercentage + 1)) / 256)
+        ; _effGasPriceShifted = gasPrice * (effectivePercentage + 1)
+        ; A => gasPrice
```

Comment:
Cosmetic: notation with `=>` is confusing here, because meaning is opposite of zkasm syntax
```suggestion
        ; A = gasPrice
```